cmd/cloud/deployment.go

```diff
@@ -134,6 +135,7 @@ func newDeploymentLogsCmd() *cobra.Command {
 	cmd.Flags().BoolVarP(&warnLogs, "warn", "w", false, "Show logs with a log level of 'warning'")
 	cmd.Flags().BoolVarP(&errorLogs, "error", "e", false, "Show logs with a log level of 'error'")
 	cmd.Flags().BoolVarP(&infoLogs, "info", "i", false, "Show logs with a log level of 'info'")
+	cmd.Flags().StringVarP(&logsKeyword, "keyword", "k", "", "Show logs that contains this keyword or phrase.")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&logsKeyword, "keyword", "k", "", "Show logs that contain this exact keyword or phrase.")
```

If no fuzzy searching, probably need to imply that it needs to be an exact match + typo